Includes/Rosetta/PlayMode/Tasks/SimpleTasks/RandomCardTask.hpp

```diff
@@ -62,6 +62,10 @@ class RandomCardTask : public ITask
     //! \return The cloned task.
     std::unique_ptr<ITask> CloneImpl() override;
 
+    static const std::vector<Card*>& GetCardList(
```

Comment:
Can you add comment for documentation?
For example,
```
    //! Processes task logic internally and returns meta data.
    //! \param player The player to run task.
    //! \return The result of task processing.
```

Reply:
And move the position of this code another method `GetCardList()` below.

Reply:
> 
> 
> And move the position of this code another method `GetCardList()` below.

It means it should be `public` method, right?

Reply:
Yes, it is.